configuration/rules-blockly-timers-and-delays.md

```diff
@@ -0,0 +1,255 @@
+---
+layout: documentation
+title: Rules Blockly - Timers & Delays
+---
+
+# Timers & Delays
+[return to Blockly Reference]({{base}}/configuration/rules-blockly.html)
+
+Timers and Delays are a little more complex but add important functionality to rules. Whilst the "Wait-For"-block is straightforward, the timer blocks should be well understood before being used - they may behave differently than expected. This chapter explains what these blocks do, sometimes displaying generated code to explain what is happening behind the scenes.
```

Comment:
Please change this to one sentence per line.
The website build will take core of a proper presentation of paragraphs.